beekeeper-integration-tests/src/test/java/com/expediagroup/beekeeper/integration/api/BeekeeperApiIntegrationTest.java

```diff
@@ -181,11 +181,107 @@ public void testGetPathsWhenThereIsFiltering() throws SQLException, InterruptedE
     Page<HousekeepingPathResponse> responsePage = mapper
         .readValue(body, new TypeReference<RestResponsePage<HousekeepingPathResponse>>() {});
     List<HousekeepingPathResponse> result = responsePage.getContent();
-
+    assertThat(responsePage.getTotalElements()).isEqualTo(1L);
     assertThatPathsEqualsResponse(testPath1, result.get(0));
     assertThat(result.size()).isEqualTo(1);
   }
 
+  @Test
+  public void testPathsPageable() throws SQLException, InterruptedException, IOException {
+    // Test the total elements on different pages by setting the page size to 1.
+    int pageSize = 1;
+
+    // Create three new HousekeepingPath objects
+    HousekeepingPath testPath1 = createHousekeepingPath("some_table",
```

Comment:
you can make these constant fields in this class if they don't change so that you dont have to copy them in all the tests